src/main/target/common_post.h

```diff
@@ -157,6 +157,7 @@
 
 #ifdef USE_SERIALRX_FPORT
 #define USE_TELEMETRY
```

Comment:
Can you please adjust these to both have #ifndef

Reply:
Done!